gatorgrouper_cli.py

```diff
@@ -48,40 +49,65 @@
 
         # generate the groups and display them
         # pylint: disable=bad-continuation
+        # If user wanted to sort by round robin and provided a group size
+        # but did not provide the number of groups
         if (
             GG_ARGUMENTS.grouping_method == constants.ALGORITHM_ROUND_ROBIN
-            and GG_ARGUMENTS.num_group is constants.DEFAULT_NUMGRP
+            and GG_ARGUMENTS.num_group is None
+            and GG_ARGUMENTS.group_size is not None
         ):
             GROUPED_STUDENT_IDENTIFIERS = group_rrobin.group_rrobin_group_size(
                 SHUFFLED_STUDENT_IDENTIFIERS, GG_ARGUMENTS.group_size
             )
+        # If the user wanted to sort by round robin and provided the number of
+        # groups but did not provide the size of the groups
         elif (
             GG_ARGUMENTS.grouping_method == constants.ALGORITHM_ROUND_ROBIN
-            and GG_ARGUMENTS.num_group is not constants.DEFAULT_NUMGRP
+            and GG_ARGUMENTS.group_size is None
+            and GG_ARGUMENTS.num_group is not None
         ):
             GROUPED_STUDENT_IDENTIFIERS = group_rrobin.group_rrobin_num_group(
                 SHUFFLED_STUDENT_IDENTIFIERS, GG_ARGUMENTS.num_group
             )
+        # If the user wanted to sort by round robin and did not provide either
+        # the number of groups or the size of the groups, group using the
+        # default number of groups
+        elif (
+            GG_ARGUMENTS.grouping_method == constants.ALGORITHM_ROUND_ROBIN
+            and GG_ARGUMENTS.group_size is None
+            and GG_ARGUMENTS.num_group is None
+        ):
+            GROUPED_STUDENT_IDENTIFIERS = group_rrobin.group_rrobin_num_group(
+                SHUFFLED_STUDENT_IDENTIFIERS, constants.DEFAULT_NUMGRP
+            )
+        # If the user wanted to sort using the graph algorithm and provided the
+        # number of groups
```

Comment:
In parse_argument.py if you set required=True, you don't need to check `args.group_size is None` and `args.num_group is None`here.